apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-docs.component.html

```diff
@@ -39,10 +39,24 @@
     <br><br>
     Buttons appearance can be changed with <code>[navigationButtonLabels]</code> property.
     <br><br>
+    If you want to use completely different style for navigational buttons,
+    you can use special directives: <code>fdpWizardGeneratorGoNextButton, fdpWizardGeneratorFinishButton</code>.
+</description>
+<component-example>
+    <fdp-wizard-generator-customizable-embeded-example></fdp-wizard-generator-customizable-embeded-example>
+</component-example>
+<code-example [exampleFiles]="customizableExample"></code-example>
```

Comment:
I guess there should be different example files as `customizableExample` appears twice in docs whether different components are shown.

Reply:
@platon-rov please check.